sync/linux/smoke-test.yaml

```diff
@@ -56,6 +56,10 @@ spec:
       labels:
         app: whoami-windows
     spec:
+      securityContext:
```

Comment:
https://github.com/antrea-io/antrea/issues/2774

workaround for `containerd github.com/containerd/containerd v1.5.5 72cec4be58a9eb6b2910f5d10f1c01ca47d231c0`

Reply:
this is failing for Calico as well. mostly some issue with containerd at this point. trying a newer version. keeping hostProcess and hostNetwork if it did not get fixed until the final solution

Reply:
Please correct me if I am wrong:
- HostProcess is Calico stuff, should be ignored by Antrea ?
- When using HostProcess require to add runAsUserName like runAsUserName: "NT AUTHORITY\\FooBar" ?

I would suggest to put this YAML in a different file and use envsubs (if calico is true add these bits)